src/frontend/context/BrevModulContext.ts

```diff
@@ -126,24 +143,7 @@ const [BrevModulProvider, useBrevModul] = createUseContext(() => {
                 avhengigheter?.brevmal.verdi !== Brevmal.VARSEL_OM_REVURDERING
             );
         },
-        avhengigheter: { brevmal },
-    });
-
-    const fritekster = useFelt<FeltState<IFritekstFelt>[]>({
```

Comment:
måtte flytte den over multiselect for å få lov til å bruke den som avhengighet...